insights/parsers/tests/test_uname.py

```diff
@@ -83,10 +85,13 @@ def test_uname():
     # Test the equality and inequality operators
     assert uname1 != uname2
     assert uname2 == uname4
+    assert uname2 == uname4_debug
     assert uname2 > uname1
     assert uname4 >= uname3
+    assert uname4_debug >= uname3
```

Comment:
If you add the debug attribute please add pos/neg tests for it as well.

Reply:
@bfahr added new tests to test weather the new attribute was true or false.